query-request.hh

```diff
@@ -366,4 +368,28 @@ public:
     friend std::ostream& operator<<(std::ostream& out, const read_command& r);
 };
 
+struct forward_request {
+    enum class reduction_type {
+        count,
+    };
+    reduction_type type;
+    std::vector<sstring> selected_columns;
```

Comment:
Why string? We should use column_id (and pass the schema id along to make sure we interpret it correctly).

Reply:
I think this is all in read_command already.